src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -59,12 +93,18 @@ private void bindListeners(ReadOnlyPerson person) {
         email.textProperty().bind(Bindings.convert(person.emailProperty()));
         person.tagProperty().addListener((observable, oldValue, newValue) -> {
             tags.getChildren().clear();
-            person.getTags().forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+            //person.getTags().forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.